runtime-sdk/src/context.rs

```diff
@@ -62,47 +152,102 @@ impl<'a> DispatchContext<'a> {
             },
             runtime_header: ctx.header,
             runtime_round_results: ctx.round_results,
-            runtime_storage: mkvs,
+            runtime_storage: storage::MKVSStore::new(ctx.io_ctx.clone(), mkvs),
+            consensus_state: &ctx.consensus_state,
             io_ctx: ctx.io_ctx.clone(),
+            logger: get_logger("runtime-sdk").new(o!("ctx" => "dispatch")),
```

Comment:
It would be cool if it also set the `mode` key to know what mode this is executing in.